cli/command/container/opts.go

```diff
@@ -12,7 +12,9 @@ import (
 	"time"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/docker/cli/cli/compose/loader"
```

Comment:
Hum although I like the fact we re-use this code, I would have extract it from there (because now `container` package depends on the `compose` package). But this is a nit 👼